src/modules/companies/companies.service.ts

```diff
@@ -59,14 +59,14 @@ export class CompaniesService {
           type: company.accounts.last_accounts?.period_start_on,
         },
         nextAccounts: {
-          dueOn: company.accounts.next_accounts.due_on,
-          overdue: company.accounts.next_accounts.overdue,
-          periodEndOn: company.accounts.next_accounts.period_end_on,
-          periodStartOn: company.accounts.next_accounts.period_start_on,
+          dueOn: company.accounts.next_accounts?.due_on,
+          overdue: company.accounts.next_accounts?.overdue,
+          periodEndOn: company.accounts.next_accounts?.period_end_on,
+          periodStartOn: company.accounts.next_accounts?.period_start_on,
         },
-        nextDue: company.accounts.next_due,
-        nextMadeUpTo: company.accounts.next_made_up_to,
-        overdue: company.accounts.overdue,
+        nextDue: company.accounts?.next_due,
```

Comment:
also worth adding a test if that property is missing since we are fixing that bug?

Reply:
@Zainzzkk The tests for this service are very confusing and fiddly - I agree a test should be added, but:

- This is difficult to achieve with the current test structure.
- We need to refactor such tests.
- Resolving this with manual tests with unblock other UKEF teams.

Reply:
@Zainzzkk The changes here do not touch `industries` / SIC codes, that's handled in `mapSicCodes` which has not changed in this PR.

 The current `mapSicCodes` implementation has a condition so that if `company.sic_codes` is undefined, an empty array will be returned.

There are edge cases of companies without SIC codes, but so far, the APIM mapping has not posed issues. If this does occur we should fix in another PR.